meta/main.yml

```diff
@@ -1,12 +1,23 @@
+---
 galaxy_info:
   author: Miroslav Lichvar <mlichvar@redhat.com>
   description: Configure NTP and/or PTP
-  galaxy_tags: [ 'system', 'time', 'redhat', 'rhel', 'fedora', 'centos', 'ntp', 'ntpd', 'chrony', 'ptp' ]
+  galaxy_tags:
+    - system
+    - time
+    - redhat
+    - rhel
+    - fedora
+    - centos
+    - ntp
+    - ntpd
+    - chrony
+    - ptp
```

Comment:
This is not necessary in my opinion. Also it's not reported as error by linter, is it?

Reply:
See also https://github.com/linux-system-roles/network/pull/108/files#diff-9b4ddf055111f4371f04fbb8b5366655

Reply:
I see. Then it would be nice to have it look the same across all linux-system-roles.

Reply:
I started from this role to see what do you think about my proposed changes. In the long run I would like to propagate those into other linux-system-roles.

Reply:
I think those proposed changes are very reasonable, just note that we have been in the process of implementing Molecule and fixing lint problems recently, so there are some open PRs, please check them first to avoid duplicating work.

Reply:
Just a note about inline, abbreviated definition of dicts and lists. This was a preferred method in ansible v1 because this is how python defines those structures. However ansible v2 puts YAML best practices higher than python ones which means that defining things (lists, dicts, module parameters) inline is still allowed, but it isn't preferred. Just to quote Ansible docs (from https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics):

> Dictionaries and lists can also be represented in an abbreviated form **if you really want to**

From my experience using inline maybe saving space and producing less lines of code but it complicates code reviews (due to how `diff` works) and prevents from fully utilizing static code analysis tools (like yamllint). Additionally it prevents IDE language server from coloring code :smile:

Reply:
> prevents from fully utilizing static code analysis tools (like yamllint)

Why? This syntax should be valid YAML, as YAML is a superset of JSON. I.e. I have been thinking that it is not because of how Python defines them, but because of JSON. Is that not the case?

Reply:
Look at those two snippets:
1)
```
- name: Generate ntp.conf file
  template: src=ntp.conf.j2 dest=/etc/ntp.conf backup=yes mode=0644
```
2)
```
- name: Generate ntp.conf file
  template:
    src: ntp.conf.j2
    dest: /etc/ntp.conf
    backup: yes
    mode: 0644
```

Both are valid yaml files but look what is the type of a `template` value. In the first snippet it is just a string (according to YAML spec) and in the second snippet it is actually a dict containing more data. Which means that first snippet cannot by fully linted as yamllint won't analyze this whole data set: `src=ntp.conf.j2 dest=/etc/ntp.conf backup=yes mode=0644` just because according to spec it a string.